packages/cli/src/commands/generate/scaffold/templates/components/Names.tsx.template

```diff
@@ -27,19 +30,20 @@ const formatEnum = (values: string | string[] | null | undefined) => {
   }
 }
 
-const truncate = (text) => {
-  let output = text
-  if (text && text.length > MAX_STRING_LENGTH) {
+const truncate = (value: string | number) => {
+  let output = value?.toString() ?? ''
+  if (value && output?.length > MAX_STRING_LENGTH) {
```

Comment:
Here's where the change I mentioned way up top would go, if you want to implement it

Reply:
I've refactored it, should make a little more sense now (hopefully)